### PR TITLE
RAIL-28 odyssey endpoints introduced

### PR DESCRIPTION
**Pull request by orzeuek**

### CODE REVIEW

File: config/common.ts [hunk 1]
```diff
@@ -183,6 +183,10 @@ export default abstract class Config {
         '^\/user\/[0-9]+': {
             name: 'customer-service',
             cacheConfig: {maxAge: 1, max: 0}
+        },
+        '^\/odyssey\/.*': {
```
Comment (orzeuek) [inline config/common.ts]: all odyssey service endpoints can have single config

File: config/common.ts [hunk 2]
```diff
@@ -183,6 +183,10 @@ export default abstract class Config {
...
+            name: 'odyssey-service',
+            cacheConfig: {maxAge: 900, max: 1000}
```
Comment (orzeuek) [inline config/common.ts]: keep in cache 15 minutes.
max: 1000 - I don't know.... just a random value from top of my head :)

**Comment by orzeuek** [comment]
@linusnorton  please review afterwards ....

File: config/common.ts (hunk 2 above)
Comment (linusnorton) [reply config/common.ts]: I think both need to be higher. I would say at least two hours and 10000

Comment (orzeuek) [reply config/common.ts]: got it. Will change - thanks :)

**Pull request merged**